xml/System.Security.Cryptography/ECDsaCng.xml

```diff
@@ -739,9 +764,25 @@
         <ReturnType>System.Security.Cryptography.KeySizes[]</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the key sizes, in bits, that are supported by the <see cref="P:System.Security.Cryptography.ECDsaCng.KeySize"/> property setter.</summary>
+        <value>An array that contains the key sizes supported by the <see cref="P:System.Security.Cryptography.ECDsaCng.KeySize"/> property setter.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+In Elliptic Curve Cryptography (ECC) the key size is not the only input into the key generation process,
+it is derived from the curve parameters for a specific elliptic curve.
+This property reports only three sizes for LegalKeySizes: 256, 384, and 521 bits.
```

Comment:
Since this is a remark for `LegalKeySizes`, maybe we can simplify this:
```suggestion
This property reports only three legal key sizes: 256, 384, and 521 bits.
```